### PR TITLE
Improve README

### LINKED ISSUE #310: Clean up README – add more sections
### What's up?

Usage section is a bit of a mess

### Mkay, tell me more...

Nothing spectacular, just adding some sections here and there

### PR DESCRIPTION
**Pull request by skidding**
Fixes #310

### CODE REVIEW

**Comment by skidding** [comment]
Added sections about integration with Create React App & Next.js

**Comment by skidding** [comment]
> does Cosmos work in CRA bootstrapped projects without ejecting the webpack config outside of node_modules/react-scripts?

Yes! Check out [Flatris](https://github.com/skidding/flatris)

**Comment by ovidiubute** [comment]
Nice! I have a few projects running on `create-react-app` but hesitated using Cosmos, I'm going to go back and add it to see how it goes 😄

**Comment by skidding** [comment]
> Nice! I have a few projects running on `create-react-app` but hesitated using Cosmos, I'm going to go back and add it to see how it goes 😄 

🙌

File: README.md [hunk 1]
```diff
@@ -202,6 +219,48 @@ We can also use [glob](https://github.com/isaacs/node-glob) instead of adding ea
 componentPaths: glob.sync('src/components/*.jsx')
 ```
 
+#### Using Create React App
```
Comment (ovidiubute) [inline README.md]: Maybe a link to `flatris` that is actually using CRA would be good?

Comment (skidding) [inline README.md]: Done 25a7bf9dab6a2cad5d764f7682ed257008fc1154

File: README.md [hunk 2]
```diff
@@ -242,7 +242,7 @@ Define `.babelrc` to enable ES6 in config & fixtures, using the CRA preset.
 
 Finally, make sure to put [proxies](#proxies) in the `src` dir, the only place included by the CRA Babel loader.
 
-CRA + Cosmos example: [Flatris](https://github.com/skidding/flatris)
+*CRA + Cosmos example: [Flatris](https://github.com/skidding/flatris)*
```
Comment (ovidiubute) [inline README.md]: 🎆

**Pull request merged**